azext_iot/deviceupdate/_help.py

```diff
@@ -260,3 +260,537 @@ def load_deviceupdate_help():
+          text: >
+            az iot device-update device class delete -n {account_name} -i {instance_name} --class-id {device_class_id} -y
+
+        - name: Delete a device class subgroup.
+          text: >
+            az iot device-update device class delete -n {account_name} -i {instance_name} --class-id {device_class_id} --group-id {device_group_id}
+    """
+
+    helps["iot device-update device group"] = """
+        type: group
+        short-summary: Device group management.
+        long-summary: >
+          A device group is a collection of devices. Device groups provide a way to scale deployments to many devices.
+          Each device belongs to exactly one device group at a time. A device group is automatically created when a Device Update-enabled
+          device is connected to the hub and reports its properties.
+    """
+
+    helps["iot device-update device group list"] = """
+        type: command
+        short-summary: List device groups within an instance.
+
+        examples:
+        - name: List device groups.
+          text: >
+            az iot device-update device group list -n {account_name} -i {instance_name}
+
+        - name: List device groups in a desired order, in this case by deviceCount.
+          text: >
+            az iot device-update device group list -n {account_name} -i {instance_name} --order-by deviceCount
+    """
+
+    helps["iot device-update device group show"] = """
+        type: command
+        short-summary: Show details about a device group including the best update and update compliance.
+
+        examples:
+        - name: Show a device group.
+          text: >
+            az iot device-update device group show -n {account_name} -i {instance_name} --group-id {device_class_id}
+
+        - name: Show the best updates available for a device group. This flag modifies the command to returns a list.
+          text: >
+            az iot device-update device group show -n {account_name} -i {instance_name} --group-id {device_group_id} --best-updates
+
+        - name: Show update compliance for a device group.
+          text: >
+            az iot device-update device group show -n {account_name} -i {instance_name} --group-id {device_group_id} --update-compliance
+    """
+
+    helps["iot device-update device group delete"] = """
+        type: command
+        short-summary: Delete a device group.
+        long-summary: >
+          Device groups are not automatically cleaned up but are retained for history purposes. This operation
+          can be used if there is no need for the group or need to retain history for it. If a device is ever connected
+          again for a group after the group was deleted it will be automatically re-created with no history.
+
+        examples:
+        - name: Delete a device group.
+          text: >
+            az iot device-update device group delete -n {account_name} -i {instance_name} --group-id {device_class_id}
```

Comment:
`--group-id {device_class_id}` should be replaced by `--group-id {device_group_id}`